jenkins_integration/jenkins_integration.php

```diff
@@ -0,0 +1,28 @@
+<?php
+//For logging in terminus
+header('Content-Type: text/plain; charset=UTF-8');
+//Create curl post request to hit the Jenkins webhook
+$secrets = json_decode(file_get_contents($_SERVER['HOME'] . '/files/private/secrets.json'));
+
+$curl = curl_init($secrets->jenkins_url);
+
+//Setup header with authentication
+curl_setopt($curl, CURLOPT_HTTPHEADER, array('Content-Type: application/json',
+    'Authorization: Basic '. base64_encode("$secrets->username:$secrets->api_token"),
+));
+
+//Declare request as a post and setup the fields
+curl_setopt($curl, CURLOPT_POST, true);
+curl_setopt($curl, CURLOPT_POSTFIELDS, array(
+  'token' => $secrets->token,
+));
+
+//Execute the request 
+$response = curl_exec($curl);
+
+if($response){
+	echo "Build Successful";
```

Comment:
$response just means that the request to start the build was successful, right?  We don't block here until the build is done, right?